### PR TITLE
Breadth first

### PR DESCRIPTION
**Pull request by nathanrhead**
- [x] Top-level README “Table of Contents” is updated
- [ ] Feature tasks for this challenge are completed
- [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
- [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/tree/tree.js [hunk 1]
```diff
@@ -110,8 +110,29 @@ class BinaryTree {
...
+    while(queue.length > 0) {
+      // let current = queue[index];
+      if (node.left) { node.left = queue[index++] }; 
+      if (node.right) { node.right = queue[index++] };
+      delete queue[0]; 
+      _traverse(queue[0]);
+
+      };
+    };
+  };
```
Comment (adrianhuebner) [inline javascript/code-challenges/tree/tree.js:135]: Nathan what you have so far looks really great! When you have a chance to come back to this, I will regrade for you!
